### PR TITLE
switch to a bit more general extra feature that is called at last responsible time before adding to zones

### PR DESCRIPTION
**Pull request by paetling**

### CODE REVIEW

**Review by ckuang — APPROVED** [review]
🕸️ 🐷

**Review by rammie — APPROVED** [review]

**Comment by rammie** [comment]
This will be a backward incompatible change that might break freshcard-app. After this lands, can we make a PR there as well that uses the renamed property?

**Comment by paetling** [comment]
Not needed atm. Holding off for now

**Pull request closed without merging**
